pkg/mqtt5/bootstrap/container.go

```diff
@@ -17,33 +17,38 @@ import (
 
 type Mqtt5ClientMap struct {
 	mqtt5Clients map[string]*mqtt5.Mqtt5Client
-	mutex        *sync.RWMutex
+	mutex        sync.RWMutex
 }
 
 // Mqtt5ClientMapName contains the name of the Mqtt5ClientMap struct in the DIC.
 var Mqtt5ClientMapName = di.TypeInstanceToName((*Mqtt5ClientMap)(nil))
 
 // Mqtt5ClientMapFrom helper function queries the DIC and returns the Dev and Remotes mode flags.
-func Mqtt5ClientMapFrom(get di.Get) Mqtt5ClientMap {
+func Mqtt5ClientMapFrom(get di.Get) *Mqtt5ClientMap {
 	mqtt5Config, ok := get(Mqtt5ClientMapName).(*Mqtt5ClientMap)
 	if !ok {
-		return Mqtt5ClientMap{}
+		return nil
 	}
 
-	return *mqtt5Config
+	return mqtt5Config
 }
 
-func (mc *Mqtt5ClientMap) Get(brokerName string) (mqtt5.Mqtt5Client, error) {
+func InitMqtt5ClientMap() Mqtt5ClientMap {
```

Comment:
since this is a factory method, use the naming convention `NewMqtt5ClientMap()`

Reply:
renamed